jujugui/views.py

```diff
@@ -74,6 +74,7 @@ def config(request):
     env_uuid = settings.get('jujugui.uuid', 'sandbox')
 
     charmstore_macaroons = settings.get('jujugui.charmstore_macaroons')
+    discharge_token = settings.get('jujugui.discharge_token')
     plans_macaroons = settings.get('jujugui.plans_macaroons')
     terms_macaroons = settings.get('jujugui.terms_macaroons')
```

Comment:
Same question about passing in macaroons for plans and terms...if we're doing non-interactive login to idm with the token, can't we let that handle all bakery requests absent passed in macaroons?